basic_pipeline/9_Sink.md

```diff
@@ -0,0 +1,69 @@
+The sink is the last element in our pipeline, designed to store the data processed by the pipeline. 
+In contrast to the filter elements, it won't have any output pad - that is why we need to make our element `use Membrane.Sink` and define the input pad only.
+Since we want to parameterize the usage of that element, it will be good to define the options structure, so that we can specify the path to the file where the output should be saved. And we also need to remember, that since the sink will work in the pipeline consisting of the elements with the pull mode typed pads, it is the sink responsibility to demand the buffer - that is why we need a `@how_much_to_demand` parameter, specifying how many buffers should be demanded each time. All of this stuff is done in the code snippet below:
```

Comment:
I would opt to remove the `@how_much_to_demand` as it blurs the image with an unnecessary parameter.

Reply:
I agree, I will remove it

Reply:
done here and in the template repo